custom-connectors/GroupMgr/apiDefinition.swagger.json

```diff
@@ -0,0 +1,325 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "GroupMgr",
+    "description": "GroupMgr is a tool used to manage your Office 365 groups.",
```

Comment:
@Jwaegebaert Please provide a bit more explanation of your connector in description as it looks very short.